frontend/src/lib/components/Annotations/annotationsLogic.ts

```diff
@@ -50,8 +50,9 @@ export const annotationsLogic = kea<annotationsLogicType<AnnotationsLogicProps>>
         annotations: {
             __default: [] as AnnotationType[],
             loadAnnotations: async () => {
+                // TODO: get rid of this
```

Comment:
What should be gotten rid of and when?

Reply:
Just the comment :)... the TODO itself got done earlier. Same for the second one.